### ISSUE TITLE
[Question] Does TTS modules (for example Tacotron2) support fine-tuning like ASR ? 

### ISSUE THREAD

**Issue by khalilRhouma** [NVIDIA/NeMo#1412]
**Is fine-tuning with TTS not supported yet?**

Am working on transfer learning with TTS models From English to the Arabic language. I saw an example of fine-tuning using ASR Quartznet model mentioned [here](https://github.com/NVIDIA/NeMo/issues/1107#issuecomment-686005952).
I am using this pre-trained model:

`tacotron2= nemo_tts.models.Tacotron2Model.from_pretrained("Tacotron2-22050Hz")`

I found that Tacotron model doesn't support  `model.change_vocabulary()`  and it raises receptions when I am using `model.setup_training_data()`.

Nemo version: 1.0.0b1
Method of NeMo install: `pip install nemo_toolkit[all]==1.0.0b1`

**Comment by donand** [comment]
Hello, any update on this issue?

I have the same problem, I would like to perform a fine-tuning of Tacotron 2 model with a different set of labels (characters/phonemes in input to the model).

I'm also interested in simple fine-tuning without changing the set of labels, is this feature already available in Nemo? I didn't find the option to start the training from a checkpoint.

Thanks!

**Comment by titu1994** [comment]
Tacotron is not easily finetunable, we suggest using other supported TTS models which support finetuning.
